glean-core/src/internal_metrics.rs

```diff
@@ -112,6 +116,15 @@ impl AdditionalMetrics {
                 },
                 TimeUnit::Millisecond,
             ),
+
+            experimentation_id: StringMetric::new(CommonMetricData {
+                name: "experimentation_id".into(),
+                category: "client.annotation".into(),
+                send_in_pings: vec!["all-pings".into()],
```

Comment:
Oh. I guess this works because our other "all-pings" metrics are actually special and handled differently internally.

We should just keep that in mind if we ever start generating code from our own metrics.yaml (we have no plans for that).

Reply:
Yeah, this wasn't _great_ but it was an already reserved "store" that the ingestion pipeline would treat the way it needed to be treated so it was fairly obvious it could be easily leveraged for the send-in-all pings purpose. I should leave a comment or document how/why it works somewhere, though.